frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/users/UserUtils.java

```diff
@@ -53,68 +61,94 @@ public UserUtils() {
         thriftClients = new ThriftClients();
     }
 
-    public static void synchronizeUserWithDatabase(User user, ThriftClients thriftClients) {
+    public static <T> com.siemens.sw360.datahandler.thrift.users.User synchronizeUserWithDatabase(T source, ThriftClients thriftClients, Supplier<String> emailSupplier, UserSynchronizer<T> synchronizer) {
```

Comment:
I find it confusing, that there is another function with the same name and the signature
```java
     public void synchronizeUserWithDatabase(User user)
```
maybe rename?